netket/vqs/mc/kernels.py

```diff
@@ -61,16 +61,23 @@ def local_value_kernel(logpsi: Callable, pars: PyTree, σ: Array, args: PyTree):
 
 
 def local_value_kernel_jax(
-    logpsi: Callable, pars: PyTree, σ: Array, O: DiscreteJaxOperator
+    logpsi: Callable, pars: PyTree, σ: Array, O: DiscreteJaxOperator, chunk_size: int,
```

Comment:
you cannot highjack functions that are around, this will break stuff.
You should just change `local_value_kernel_jax_chunked` below..

Reply:
Ahh I see, I will write a new function

Reply:
thanks